ssp/environment.go

```diff
@@ -35,6 +36,7 @@ type Environment struct {
 	BaseEnvironment             *Environment `jsonapi:"relation,base_environment"`
 	Usage                       Usage
 	PHPVersion                  string `jsonapi:"attr,php_version"`
+	InstanceType                string `jsonapi:"attr,instance_type",omitempty`
```

Comment:
omitempty outside of the "? check https://github.com/google/jsonapi for the correct syntax

Reply:
Oops, fixed